xml/hwbp_registers.xml

```diff
@@ -277,14 +277,15 @@ same project unless stated otherwise.
         `hstateenzero[57]` in the Smstateen extension.  Enabling {csr-scontext}
         can be a security risk in a virtualized system with a hypervisor that
         does not swap {csr-scontext}.
-        <field name="data" bits="XLEN-1:0" access="WARL" reset="0">
+        <field name="0" bits="XLEN-1:32" access="R" reset="0" />
+        <field name="data" bits="31:0" access="WARL" reset="0">
             Supervisor mode software can write a context number to this
             register, which can be used to set triggers that only fire in that
             specific context.
 
             An implementation may tie any number of high bits in this field to
             0. It's recommended to implement no more than 16 bits on RV32, and
-            34 on RV64.
+            32 on RV64.
```

Comment:
Now I notice that this recommendation is confusing because it's not possible to implement more than 32 bits.  We can just drop everything after the comma.

Reply:
I had read this as:
1. It's recommended to implement no more than 16 bits on RV32.
2. It's recommended to implement 32 bits on RV64.

But I agree it's ambiguous.

Maybe we should scrap the "no more" altogether, and just recommend 16 and 32? It's already clear that fewer are allowed.

E.g.
> It's recommended to implement 16 bits on RV32, and 32 on RV64.

Reply:
I'm OK with your proposal but I'd eliminate the comma and then maybe say "bits" to be more symmetric: "It's recommended to implement 16 bits on RV32 and 32 bits on RV64."